### PR TITLE
adding stat + 404 + autopep8

### PR DESCRIPTION
**Pull request by MounaBenAli**
I have not been able to test my code locally since Flask could not be started :

This is the error message I received when running this : 
``` HBNB_MYSQL_USER=hbnb_dev HBNB_MYSQL_PWD=hbnb_dev_pwd HBNB_MYSQL_HOST=localhost HBNB_MYSQL_DB=hbnb_dev_db HBNB_TYPE_STORAGE=db HBNB_API_HOST=0.0.0.0 HBNB_API_PORT=5000 python3 -m api.v1.app```

Error : 
```ImportError: cannot import name 'app_views' from partially initialized module 'api.v1.views' (most likely due to a circular import) (/home/mona/AirBnB_clone_v3/api/v1/views/__init__.py) ```

### CODE REVIEW

**Comment by cryptolake** [comment]
The error was because of autopep8 it put `from api.v1.views.index import *`  in the beginning of the file `api/v1/views/__init__.py`

File: api/v1/views/__init__.py [hunk 1]
```diff
@@ -2,9 +2,8 @@
 """
 Initializer
 """
+from api.v1.views.index import *
 from flask import Blueprint
 
 
 app_views = Blueprint("app_views", __name__, url_prefix="/api/v1")
-
-from api.v1.views.index import *
```
Comment (cryptolake) [inline api/v1/views/__init__.py:10]: this line is one causing the problem

Comment (cryptolake) [inline api/v1/views/__init__.py:10]: this is the one causing the problem

Comment (MounaBenAli) [reply api/v1/views/__init__.py]: I understand now ty

File: models/engine/file_storage.py [hunk 2]
```diff
@@ -48,36 +46,36 @@ def count(self, cls=None):
...
                 jo = json.load(f)
             for key in jo:
                 self.__objects[key] = classes[jo[key]["__class__"]](**jo[key])
-        except:
+        except Exception:
             pass
 
     def delete(self, obj=None):
-        """delete obj from __objects if it’s inside"""
+        """Delete obj from __objects if it’s inside."""
```
Comment (cryptolake) [inline models/engine/file_storage.py:73]: Always format comments like this: first word needs to be in imperative, and capitalized.
And if it's one comment it should be with the """ on one line.

**Review by cryptolake — APPROVED** [review]

**Pull request merged**